pytket/docs/index.rst

```diff
@@ -1,19 +1,20 @@
 pytket
 ======
 
-.. image:: CQCLogo.png
-   :width: 120px
+.. image:: Quantinuum_logo.png
+   :width: 200px
    :align: right
 
-``pytket`` is a python module for interfacing with `CQC`_ tket, a set of quantum
-programming tools. We currently support circuits and device architectures from
+``pytket`` is a python module for interfacing with tket, a quantum computing toolkit and optimising compiler developed by Quantinuum. We currently support circuits and device architectures from
```

Comment:
```suggestion
``pytket`` is a python module for interfacing with tket, a quantum computing toolkit and optimising compiler developed by `Quantinuum`_. We currently support circuits and device architectures from
```

I think you need to do this to link it to the URL below?

Reply:
Ah thanks, should be fixed now.